### PR TITLE
Update functions.md

### PR DESCRIPTION
**Pull request by Ivo-Balbaert**
array.delete should take index instead of element - code corrected so that it works and gives the output as printed

### CODE REVIEW

File: examples/arrays/functions.md [hunk 1]
```diff
@@ -48,15 +48,15 @@ Output
...
-Deletes the element present in the array.
+Deletes the element present in the array at index ix.
 
 ```go
-even_numbers = [2, 4, 6, 8, 10]
-even_numbers.delete(8)
-println(even_number)
+mut even_numbers = [2, 4, 6, 8, 10]
+even_numbers.delete(3)
+println(even_numbers)
```
Comment (vbrazo) [inline examples/arrays/functions.md:59]: 👍

Comment (dhonx) [reply examples/arrays/functions.md:59]: great  👍

File: examples/primitives/primitives.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Primitives
 
-V has less number of primitive types like Go.
+V has less primitive types than Go.
```
Comment (donnisnoni95) [inline examples/primitives/primitives.md]: Great

File: examples/arrays/functions.md [hunk 3]
```diff
@@ -48,15 +48,15 @@ Output
 Syntax
 
 ```go
-array.delete(element T)
+array.delete(ix int)
 ```
 
-Deletes the element present in the array.
+Deletes the element present in the array at index ix.
```
Comment (dhonx) [inline examples/arrays/functions.md]: maybe look nice if we highlighted the `ix`

**Review by dhonx — APPROVED** [review]

**Pull request merged**